АКОС ДЗ (Модуль напоминаний)

ask4isn opened the pull request.

ask4isn committed:
lab

ask4isn commented:
Аманов Азамат (Б05-252)

Andy-Messer reviewed (changes requested):
Необходимо добавить механизмы [синхронизации](https://habr.com/ru/articles/278413/) кода. Мне маловато документации, надо описать, как пользоваться модулем. Желательно так же указать максимальную длину сообщения. 

исправь синхронизацию и будет отл, пока что хор 7

ask4isn committed:
spinlock + README.md

- Добавил spinlock там, где он нужен, для синхронизации
- Так как добавляем ограничение на макс. длину сообщении, также выделил и макс. число событии
- Постарался добавить уточняющей документации

Andy-Messer commented:
Хорошо, мне нравится, ставлю 9
